server/images.go

```diff
@@ -471,6 +472,14 @@ func paramsToReader(params map[string]string) (io.ReadSeeker, error) {
 					field.SetBool(boolVal)
 				case reflect.String:
 					field.SetString(val)
+				case reflect.Slice:
+					re := regexp.MustCompile(`"(.*?)"`) // matches everything enclosed in quotes
+					vals := re.FindAllStringSubmatch(val, -1)
```

Comment:
It seems you're using regex to get all the values wrapped in double quotes? This likely won't work since the parser parses out a single parameter value from the line

e.g.

This is parsed as `parameters[stop]=[]string{"foo"}`
```
PARAMETER stop "foo" "bar" "baz"
```

While this is parsed as `parameters[stop]=[]string{"foo bar baz"}`
```
PARAMETER stop foo "bar baz"
```

Clearly this is the wrong behaviour but implementing a parser that can handle this correctly is not worthwhile right now. 

What I have in mind is something like this to create `parameters[stop]=[]string{"foo", "bar baz"}`

```
PARAMETER stop foo
PARAMETER stop "bar baz"
```

Reply:
good suggestion, doing this instead

Reply:
made this change